modules/apps/object/object-api/src/main/java/com/liferay/object/exception/ObjectRelationshipEdgeException.java

```diff
@@ -15,8 +15,10 @@ public class ObjectRelationshipEdgeException extends PortalException {
 	public ObjectRelationshipEdgeException() {
 	}
 
-	public ObjectRelationshipEdgeException(String msg) {
+	public ObjectRelationshipEdgeException(String msg, String messageKey) {
```

Comment:
Hey @victorKammerer could you please rename `String msg` to `String message` to keep the pattern used in other exceptions like this [one](https://github.com/liferay/liferay-portal/blob/1cfd18f9cbc0e7be4f28ccd2da369ed6f0b1620d/modules/apps/object/object-api/src/main/java/com/liferay/object/exception/ObjectActionNameException.java#L76) for example.

Thank you!

Reply:
Done!